src/config/config.go

```diff
@@ -64,11 +64,11 @@ func ValidDateConfiguration(config *Configuration) {
 			fmt.Println("Please enter a valid pathway to the static file")
 		}
 	} else if fileInputL != "live" && fileInputL != "static" {
-		fmt.Println("Please enter either live or static as a dataOption in config.json")
+		fmt.Println("Please enter either live or static as a dataOption in default-config.json")
 	} else if outlierL == "mad" {
 		//require mad parameter
```

Comment:
Should there be something here if the "outlierL == mad"?

Reply:
^Jolene's next pull request